### ISSUE TITLE
Tools that create only collections do not show any errors in the users history

### ISSUE THREAD

**Issue by lparsons** [galaxyproject/galaxy#3554]
If a tool that creates only collections (e.g. `toolshed.g2.bx.psu.edu/repos/iuc/trinity_run_de_analysis/trinity_run_de_analysis/2.2.0.1`) ends with an error, the user's history contains only green collection boxes with no dataset. This means:

1. They are unable to see that there was an error (though the empty collections are a tipoff).
2. They are unable to see the error message, and thus unlikely to be able to fix the problem.

**Comment by lparsons** [comment]
Ping @jmchilton, the collections guy. ;-)

**Comment by jmchilton** [comment]
Yeah - this is definitely a problem - we convey this information only through dataset currently. I've started to think about this - I'll ping you if I make any progress.

**Comment by jpungor** [comment]
I am having a similar issue, I ran Trinity on two fastq.gz files and it unzipped them both, but returned an empty assembly with no errors reported in the log. Does anyone know what to do about that?
